controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ResourceMeterMaintainer.java

```diff
@@ -85,9 +87,23 @@ protected void maintain() {
                                 )).values();
     }
 
-    private Predicate<Node> unlessNodeOwnerIsHostedVespa() {
-        return node -> node.owner().map(owner ->
-                !owner.tenant().value().equals("hosted-vespa")
-        ).orElse(false);
+    private boolean unlessNodeOwnerIsHostedVespa(Node node) {
+        return node.owner()
+                .map(owner -> !owner.tenant().value().equals("hosted-vespa"))
+                .orElse(false);
+    }
+
+    /**
+     * Checks if the node is in some state where it is in active use by the tenant,
+     * and not transitioning out of use, in a failed state, etc.
+     */
+    private static final Set<Node.State> METERABLE_NODE_STATES = Set.of(
+            Node.State.reserved,   // an application will soon use this node
+            Node.State.active,     // an application is currently using this node
+            Node.State.inactive    // an application owner might set a node inactive
```

Comment:
Nitpick, but I believe application owner has no control over deactivation of nodes?

Reply:
Correct, moved to inactive by configserver when the node is no longer needed (retired manually, requested flavor changed, number of nodes changed).

Reply:
Aha! But they are assigned to a user, and just waiting for clean-up of the node?

Reply:
Yes, they are assigned to the user and just waiting to be expired, the purpose of that state is if the user deploy unintended change and starts losing nodes, they can undo it without losing data (the inactive nodes will be assigned back to them)